visma/solvers/simulEqn.py

```diff
@@ -27,20 +28,34 @@ def coeffCalculator(LandR_tokens):
             )
         for _, token in enumerate(lTokens):
             if isinstance(token, Variable):
-                if token.value == ['x']:
+                if token.value == [variables[0]]:
                     coefficients[i][0] = token.coefficient
-                elif token.value == ['y']:
+                elif token.value == [variables[1]]:
                     coefficients[i][1] = token.coefficient
-                elif token.value == ['z']:
+                elif token.value == [variables[2]]:
                     coefficients[i][2] = token.coefficient
             if isinstance(token, Constant):
                 coefficients[i][3] = token.value
 
     return coefficients
 
 
-def getResult(matD, matDx, matDy, matDz):
-    '''Calculates values of x, y and z'''
+def getResult(matD, matDx, matDy, matDz, variables, comments, animation, solveFor):
+    '''Calculates values of x, y and z
+
+    Arguments:
+    matD {visma.matrix.structure.Matrix.SquareMat} -- Matrix Token
```

Comment:
```suggestion
        matD {visma.matrix.structure.Matrix.SquareMat} -- Matrix Token
```
Same in `simulSolver` docstring.